fili-core/src/main/java/com/yahoo/bard/webservice/config/DefaultLogicalTableInfo.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config;
+
+/**
+ * Default for LogicalTableInfo interface, everything just returns getName.
+ */
+@FunctionalInterface
+public interface DefaultLogicalTableInfo extends LogicalTableInfo {
+
```

Comment:
I think perhaps we should go ahead and make this a class that constructs off String name.